Code/GraphMol/FileParsers/FileParsers.h

```diff
@@ -286,6 +286,17 @@ RDKIT_FILEPARSERS_EXPORT RWMol *Mol2BlockToMol(
     const std::string &molBlock, bool sanitize = true, bool removeHs = true,
     Mol2Type variant = CORINA, bool cleanupSubstructures = true);
 
+
+RDKIT_FILEPARSERS_EXPORT void ParseExtraLine(std::string extraLine);
+RDKIT_FILEPARSERS_EXPORT Atom *ParseXYZFileAtomLine(std::string atomLine, RDGeom::Point3D &pos, unsigned int line);
+RDKIT_FILEPARSERS_EXPORT RWMol *XYZDataStreamToMol(std::istream &inStream);
+// \brief construct a molecule from an xyz file
+/*!
+ *   \param fName    - string containing the file name
+ *   \param charge - the integer charge of the molecule
+ */
+RDKIT_FILEPARSERS_EXPORT RWMol *XYZFileToMol(const std::string &fName, int charge=0);
```

Comment:
Still also need a declaration of `XYZBlockToMol` in order to be consistent with the other file parsers.